exampleSite/utilities/examples/data/events/yyyy-city.yml

```diff
@@ -46,6 +52,11 @@ team_members: # Name is the only required field for team members.
     twitter: "devopsdays"
   - name: "Sally Fields"
     employer: "Acme Anvil Co."
+    github: "devopsdays"
+    facebook: "https://www.facebook.com/sally.fields"
+    linkedin: "https://www.linkedin.com/in/sallyfields"
+    website: "http://devopsdays.org"
```

Comment:
Would a different sample site make it more obvious that this is intended to be a link to the organizer's offsite presence?